tests/unit/cmd/iac/test_scan_common.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+from typing import List
+
+import pytest
+from click.testing import CliRunner
+
+from ggshield.cmd.main import cli
+from tests.repository import Repository
+from tests.unit.conftest import _IAC_SINGLE_VULNERABILITY, my_vcr
+
+
+COMMANDS = [["scan"], ["scan", "all"], ["scan", "diff", "--ref", "HEAD~1"]]
+
+
+@my_vcr.use_cassette("test_iac_scan_exit_zero")
```

Comment:
Isn't there an issue with the cassettes here ?
Here it will use the same cassette for each parameter of the parametrize.
This means that two of your cassettes above are useless.

What you can do is use `my_vcr` as a context manager and add the name of the cassette in the parameters. (You should be able to use the same cassette when exit zero is given and when it is not.)